src/cascade/executor/epiviz_runner.py

```diff
@@ -131,6 +158,10 @@ def model_context_from_settings(execution_context, settings):
     add_mortality_data(model_context, execution_context)
     add_omega_constraint(model_context, execution_context)
```

Comment:
I guess both of these "add's" update "model_context.input_data.observations"
Are "mtspecific" and "mtall" being added to observations? 
Maybe: add_cause_specific_mortality_data_to_observations?
Maybe: add_mtall_and_omega-constraint_to_observations?
Or maybe follow the same approach as for bundle_to_observations?
so that, model_context.input_data.observations = observations_with_mtspecific?